### DISCUSSION TITLE
[WP241 Discussion]: Serine without connections

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#50] [General]
### Discussion

Serine is not connected to anything in the pathway. What is its role?

### Should this pathway be deleted?

- [ ] Pathway should be deleted

**Comment by KestUrba** [comment]
Amino acid Serine found in a food or synthesised in the branch of glycolysis forms the input into One Carbon Units metabolism. Ser (three carbons) conversion to Glycine (two carbons) must be presented clearly as the rest one carbon could form DNA nucleotides (left direction) or perform DNA methylation (right direction) or support polyamines synthesis... This conversion could be noticed at the  ROCHE pathways map B7 square [http://biochemical-pathways.com/#/map/1](url)
